crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -124,24 +126,87 @@ where
     M::Data: PartialEq + Eq + Hash + Clone,
 {
     fn build(&self, app: &mut bevy_app::App) {
+        let no_prepass_plugin_loaded = app.world.get_resource::<AnyPrepassPluginLoaded>().is_none();
+
+        if no_prepass_plugin_loaded {
+            app.insert_resource(AnyPrepassPluginLoaded);
+
+            // At the start of each frame, last frame's GlobalTransforms become this frame's PreviousGlobalTransforms
+            app.add_system(update_mesh_previous_global_transforms.in_base_set(CoreSet::PreUpdate));
```

Comment:
```suggestion
            app.add_system(update_mesh_previous_global_transforms.in_base_set(CoreSet::PreUpdate));
            app.add_system(update_previous_view_projections.in_base_set(CoreSet::PreUpdate));
```

the prev camera update looks to have been dropped by mistake at some point

Reply:
Ahh good catch, idk how I keep making that mistake :sweat_smile: .